.storybook/utils/resetGlobals.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Storybook utilities.
```

Comment:
Let's update this to be specific to the utility since the name implies that it only provides a single utility.

Reply:
👍